xwiki-platform-core/xwiki-platform-ratings/xwiki-platform-ratings-ui/src/main/resources/XWiki/Ratings.xml

```diff
@@ -446,20 +446,22 @@ position: relative;
     </class>
     <property>
       <content>{{velocity}}
-#set($canView = $services.security.authorization.hasAccess("view", "XWiki.RatingsMacros"))
+#set($canView = $services.security.authorization.hasAccess('view', 'XWiki.RatingsMacros'))
 #if($canView)
   {{include reference='XWiki.RatingsMacros'/}}
-  #set($configurationClass = 'XWiki.RatingsConfigClass')
+#end
+{{/velocity}}
```

Comment:
I see, the second macro needs the `include` to be executed first. Is it terrible if the include happens anyway, i.e. move the include macro outside, before the velocity macro?